docs/design/README.md

```diff
@@ -213,10 +371,26 @@ The behavior of the Carbon compiler depends on the _build mode_:
 
 Expressions compute values in Carbon, and these values are always strongly typed
 much like in C++. However, an important difference from C++ is that types are
-themselves modeled as values; specifically, compile-time constant values. This
-means that the grammar for writing a type is the [expression](#expressions)
-grammar. Expressions written where a type is expected must be able to be
-evaluated at compile-time and must evaluate to a type value.
+themselves modeled as values; specifically, compile-time-constant values. This
+has a number of consequences:
+
+-   Names for types are in the same namespace shared with functions, variables,
+    namespaces, and so on.
+-   The grammar for writing a type is the [expression](#expressions) grammar,
+    not a separate grammar for types. So Carbon doesn't use angle brackets
+    `<`...`>` in types, since `<` and `>` are used for comparison in
+    expressions.
```

Comment:
Not sure if I'm finding the best grammar tweak here, but maybe:

```suggestion
    not a separate grammar for types. Carbon also doesn't use angle brackets
    `<`...`>` in types, since `<` and `>` are used for comparison in
    expressions.
```

Reply:
Ok, I worry that folks may think that is the *only* reason, but I don't think it's a big deal either way. The wording in the latest version seems fine.